src/dashboard/src/components/accounts/views.py

```diff
@@ -62,6 +65,37 @@ def add(request):
     })
 
 
+def profile(request):
+    if get_client_config_value('kioskMode') == 'True':
+        return redirect('main.views.forbidden')
```

Comment:
ah yes, I remember seeing that removed in your config pull request. This was based on an older version of the code so I'll be sure to double check this when rebasing.